tests/snapshot-test/flat/next/snapshot.test.js

```diff
@@ -1,12 +1,23 @@
+/* eslint-disable unicorn/prefer-module */
 const { getESLintConfig } = require('../../helper');
 
-test('should match ESLint Flat Configuration snapshot: next', async () => {
-  const filePath = './dummy.page.tsx';
-  const { languageOptions, rules, settings } = await getESLintConfig(
-    filePath,
-    __dirname,
-    true,
-  );
+describe('ESLint Configuration Snapshot Tests', () => {
+  beforeAll(() => {
+    process.env.TSCONFIG_ROOT_DIR = '/dummy';
+  });
```

Comment:
A dummy path is assigned to `process.env.TSCONFIG_ROOT_DIR` only during snapshot test to prevent `import.meta.dirname` from being referenced.